src/rust/ide/src/view/node_editor.rs

```diff
@@ -701,25 +708,36 @@ impl GraphEditorIntegratedWithControllerModel {
     }
 
     fn node_entered_in_ui(&self, node_id:&graph_editor::NodeId) -> FallibleResult<()> {
+
         debug!(self.logger,"Requesting entering the node {node_id}.");
-        let id           = self.get_controller_node_id(*node_id)?;
-        let controller   = self.controller.clone_ref();
-        let logger       = self.logger.clone_ref();
-        let enter_action = async move {
+        let id             = self.get_controller_node_id(*node_id)?;
+        let info           = self.lookup_computed_info(&id);
+        let controller     = self.controller.clone_ref();
+        let logger         = self.logger.clone_ref();
+        let graph_editor   = self.editor.clone_ref();
+        let enter_action   = async move {
             let result = controller.enter_node(id).await;
             debug!(logger,"Entering node result: {result:?}.");
+            let method_pointer = info.as_ref().and_then(|info| info.method_pointer.as_ref());
+            if let (Ok(_),Some(method_pointer)) = (result,method_pointer) {
+                graph_editor.breadcrumbs.frp.push_breadcrumb.emit(&method_pointer.name)
+            }
```

Comment:
Please, log error when there is no `info`. Also the current debug log in 720 should be an error log in case we weren't able to enter node at all.